services-monitoring/prometheus/prometheus.yml

```diff
@@ -66,5 +66,7 @@ scrape_configs:
         target_label: __param_target
       - source_labels: [__param_target]
         target_label: instance
+      - target_label: device
+        replacement: alacer
```

Comment:
I think making it target-specific is a better option. Something like this in `static_configs`
```
      - targets:
        - '10.12.5.2'
        labels:
          device: alacer
```